### ISSUE TITLE
Unsatisfiable Error Clarification

### ISSUE THREAD

**Issue by sim0nlee** [pyg-team/pytorch_geometric#8161]
Hi,

I am attempting to install PyG through Conda and would like a clarification on the meaning of the following error I keep encountering:

```
(dl) simo@simone-ubuntu:~$  conda install pyg -c pyg
Collecting package metadata (current_repodata.json): done
Solving environment: unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Solving environment: unsuccessful attempt using repodata from current_repodata.json, retrying with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Solving environment: | 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                              

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versionsThe following specifications were found to be incompatible with your system:

  - feature:/linux-64::__cuda==12.2=0
  - feature:/linux-64::__glibc==2.35=0
  - feature:|@/linux-64::__cuda==12.2=0
  - feature:|@/linux-64::__glibc==2.35=0
  - pyg -> pytorch=2.0 -> __cuda[version='>=11.8']

Your installed version is: 2.35
```

### Environment

* PyG version: -
* PyTorch version: 2.1.0
* OS: Ubuntu 22.04.3 LTS
* Python version: 3.11.5
* CUDA/cuDNN version: 12.1 according to PyTorch but 12.2 according to nvcc --version
* How you installed PyTorch and PyG (`conda`, `pip`, source): `conda`
* Any other relevant information (*e.g.*, version of `torch-scatter`):

**Comment by simo-tuomisto** [comment]
I've encountered the same error. It seems that pyg built against torch 2.1 is missing from pyg-channel:

https://anaconda.org/pyg/pyg/files

In addition, pyg_lib wheel is missing from these directories:

https://data.pyg.org/whl/torch-2.1.0+cu118.html
https://data.pyg.org/whl/torch-2.1.0+cu121.html

**Comment by rusty1s** [comment]
Yes, we will release PyG 2.4 and `pyg-lib==0.3.0` soon (probably tomorrow).

**Comment by sim0nlee** [comment]
I solved the problem by installing PyG with ```pip``` as @rusty1s said and I removed ```pyg_lib``` from the command to install the dependencies. If this is a good solution I can close the issue.

**Comment by rusty1s** [comment]
Ok. We also just released `pyg-lib==0.3.0`, so I think this issue can be closed.
